bootstrap/service.go

```diff
@@ -380,13 +397,36 @@ func (bs bootstrapService) DisconnectThingHandler(ctx context.Context, channelID
 	return nil
 }
 
-func (bs bootstrapService) identify(ctx context.Context, token string) (string, error) {
+func (bs bootstrapService) identify(ctx context.Context, token string) (*magistrala.IdentityRes, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Second)
 	defer cancel()
 
 	res, err := bs.auth.Identify(ctx, &magistrala.IdentityReq{Token: token})
 	if err != nil {
-		return "", errors.Wrap(svcerr.ErrAuthentication, err)
+		return nil, svcerr.ErrAuthentication
+	}
+	if res.GetId() == "" || res.GetDomainId() == "" {
+		return nil, svcerr.ErrAuthentication
+	}
+	return res, nil
+}
+
+func (bs bootstrapService) authorize(ctx context.Context, domainID, subjType, subjKind, subj, perm, objType, obj string) (string, error) {
```

Comment:
If authorize is being called with the same parameters we can reduce the number of parameters and use constants for example 
```diff
- Permission: perm,
+ Permission: auth.MembershipPermission,
```